subsys/net/l2/openthread/openthread.c

```diff
@@ -487,11 +493,11 @@ static int openthread_init(struct net_if *iface)
 
 	openthread_api_mutex_unlock(ot_context);
 
-	ot_tid = k_thread_create(&ot_thread_data, ot_stack_area,
-				 K_KERNEL_STACK_SIZEOF(ot_stack_area),
-				 openthread_process, ot_context, NULL, NULL,
-				 OT_PRIORITY, 0, K_NO_WAIT);
-	k_thread_name_set(&ot_thread_data, "openthread");
+	k_work_queue_start(&ot_context->work_q, ot_stack_area,
+			   K_KERNEL_STACK_SIZEOF(ot_stack_area),
+			   OT_PRIORITY, &q_cfg);
+
+	(void)k_work_submit_to_queue(&ot_context->work_q, &ot_context->api_work);
```

Comment:
nit:
```suggestion
	ARG_UNUSED(k_work_submit_to_queue(&ot_context->work_q, &ot_context->api_work));
```

Reply:
I didn't find any other example of `ARG_UNUSED` in combination with a function call...

Reply:
Yes, not a big deal. It's just a decorator which tells more than just `(void)`.

Reply:
Yes, but the decorator also specifies that it is targeted for an argument. If for some reason `ARG_UNUSED` on some platform or in a future update does something different than simply `(void)`, it could break.